.env.example

```
@@ -27,4 +27,7 @@ SENTRY_AUTH_TOKEN=
 # PostHog Configuration
 NEXT_PUBLIC_POSTHOG_KEY=
 NEXT_PUBLIC_POSTHOG_HOST=
-NEXT_PUBLIC_POSTHOG_ENABLED=
\ No newline at end of file
+NEXT_PUBLIC_POSTHOG_ENABLED=
+
+# Google Analytics
+NEXT_PUBLIC_GA_MEASUREMENT_ID=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)